src/App.tsx

```diff
@@ -5,28 +5,51 @@ import * as RLP from "rlp";
 type AppState = {
   mode: "encoded" | "decoded";
   text: string;
-  rlp: any;
+  rlp: {
+    valid: boolean;
+    decoded: any;
+  };
 };
 
 class App extends React.Component<{}, AppState> {
   public constructor() {
     super({});
+
+    const stock = "0xc701c502c303c104";
```

Comment:
What is it?

Reply:
I want to put there a placeholder. It is a RLP encoded value of `[1,[2,[3,[4]]]]`.

Reply:
How about using `placeholder` attribute if you want a placeholder?